### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by RolinAnn**

### CODE REVIEW

File: lab-exercise-1/AWLFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  public class AWLFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (TreephobKho) [inline lab-exercise-1/AWLFeature.java:3]: Please include this part.

Comment (Pakin-aw) [inline lab-exercise-1/AWLFeature.java:3]: Nice comment.

File: lab-exercise-1/AWLFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+  public class AWLFeature {
+      public void feature() {
```
Comment (ACitronella) [inline lab-exercise-1/AWLFeature.java:2]: nice feature

File: lab-exercise-1/AWLFeature.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+  public class AWLFeature {
```
Comment (BunradarCH) [inline lab-exercise-1/AWLFeature.java:1]: good feature

File: lab-exercise-1/AWLFeature.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+      public void feature() {
+          // this is a new feature
+      }
```
Comment (sorrymaker1130) [inline lab-exercise-1/AWLFeature.java:4]: plz add more function in the code

File: lab-exercise-1/AWLFeature.java (hunk 1 above)
Comment (RolinAnn) [inline lab-exercise-1/AWLFeature.java:3]: Nice feature.

File: lab-exercise-1/AWLFeature.java [hunk 5]
```diff
@@ -1,5 +1,7 @@
   public class AWLFeature {
       public void feature() {
           // this is a new feature
+        int x = 0;
+        x = x + 1;
```
Comment (TreephobKho) [inline lab-exercise-1/AWLFeature.java:5]: Thank you for adding this part.

**Comment by RolinAnn** [comment]
The new fix is available now.

**Comment by RolinAnn** [comment]
The new fix is available now.

File: lab-exercise-1/AWLFeature.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+  public class AWLFeature {
+      public void feature() {
+          // this is a new feature
+        int x = 0;
+        x = x + 1;
```
Comment (sorrymaker1130) [inline lab-exercise-1/AWLFeature.java:5]: Add the part successful.

File: lab-exercise-1/AWLFeature.java [hunk 7]
```diff
@@ -0,0 +1,7 @@
+  public class AWLFeature {
+      public void feature() {
+          // this is a new feature
+        int x = 0;
```
Comment (Pakin-aw) [inline lab-exercise-1/AWLFeature.java:4]: Nice

File: lab-exercise-1/AWLFeature.java (hunk 6 above)
Comment (RolinAnn) [inline lab-exercise-1/AWLFeature.java:5]: Thank you for adding this part.

File: lab-exercise-1/AWLFeature.java [hunk 8]
```diff
@@ -0,0 +1,7 @@
+  public class AWLFeature {
```
Comment (ACitronella) [inline lab-exercise-1/AWLFeature.java:1]: nice

Comment (BunradarCH) [inline lab-exercise-1/AWLFeature.java:1]: good

**Pull request merged**